src/components/core/tabs/tabs.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactElement, useEffect, useRef, useState } from 'react';
+import type { HtmlHTMLAttributes } from 'react';
+import classNames from 'classnames';
+import type { TabProps } from './tab';
+import Tab from './tab';
+import css from './tabs.module.scss';
+
+export interface TabsProps extends HtmlHTMLAttributes<HTMLDivElement> {
+    active_index?: number;
+    contained?: boolean;
+    dark?: boolean;
+    children?: ReactElement<TabProps> | ReactElement<TabProps>[];
```

Comment:
```suggestion
    children?: ReactElement<TabProps>[];
```
this will cause map as undefined when you try to consume it.

Reply:
i used `React.Children.map` instead